src/main/java/uk/gov/justice/hmpps/prison/repository/MovementsRepository.java

```diff
@@ -71,13 +72,14 @@
     }
 
 
-    public List<Movement> getMovementsByOffenders(final List<String> offenderNumbers, final List<String> movementTypes, final boolean latestOnly) {
+    public List<Movement> getMovementsByOffenders(final List<String> offenderNumbers, final List<String> movementTypes, final boolean latestOnly, final boolean allBookings) {
```

Comment:
Could we change `latestOnly` to `latestMovementOnly`?  Otherwise it's confusing me as the parameters sound like they conflict.